tests/handlers/test_room_summary.py

```diff
@@ -1092,3 +1092,29 @@ def test_visibility(self):
         )
         result = self.get_success(self.handler.get_room_summary(user2, self.room))
         self.assertEqual(result.get("room_id"), self.room)
+
+    def test_fed(self):
+        """
+        Return data over federation and ensure that it is handled properly.
+        """
+        fed_hostname = self.hs.hostname + "2"
+        fedroom = "#subroom:" + fed_hostname
```

Comment:
nit: For consistency with our naming convention, this ought to be `fed_room`

Reply:
Right, I renamed it from subroom, which for some reason didn't use an underscore :3